admin_guide/assembly_restore-etcd-quorum.adoc

```diff
@@ -13,9 +13,13 @@
 toc::[]
 
 
-If you lose etcd quorum, you must back up etcd, take down your etcd cluster,
-and form a new one. You can use one healthy etcd node to form a new cluster, 
-but you must remove all other healthy nodes.
+If you lose etcd quorum, you can restore it.
+
+* If you run etcd as a separate service, you must back up etcd, take down
+your etcd cluster, and form a new one. You can use one healthy etcd node to form
+a new cluster, but you must remove all other healthy nodes.
+* If you run etcd as static pods on your master nodes, you stop the etcd pods,
```

Comment:
I believe etcd only runs as a static pod as of 3.10, and am not aware of the user having the option to run as a service

Reply:
cc @mfojtik does the user have this option?

Reply:
yes, there is no option for that

Reply:
I thought @sdodson said that if you already had an etcd cluster and upgraded to 3.10, you kept your current etcd architecture instead of getting converted to static pods. You also have the option of putting etcd on separate nodes and running it as a separate service, there.

Do you agree with those two scenarios? If so, how would you briefly describe them?

Reply:
That's correct, if upgrading from 3.9 with separate etcd hosts those will remain in place. Perhaps s/separate service/separate hosts/ ?

Reply:
Got it. I think that your suggestion would be perfect if you could run it only on separate nodes, so I added more details at  line 16.

Reply:
Scott clarified more in scrum - if you were running master as a separate service on a master node it **is** converted to a static pod. I'm going to make more updates.